src/nvim/edit.c

```diff
@@ -4342,6 +4364,16 @@ ins_compl_next (
   return num_matches;
 }
 
+void pum_external_select_item(int item, bool insert, bool finish) {
```

Comment:
`pum_ext_select_item` . We don't use "external" AFAICT. It's nice that "ext" means "extension" also, since "external" does not always make sense and we're more interested in extensibility.

N.B. I would propose renaming `ui_is_external` to `ui_has`, but waiting on the big PRs to merge first.

Reply:
> N.B. I would propose renaming ui_is_external to ui_has, but waiting on the big PRs to merge first.

Fine by me. Just do it if you want it to happen, otherwise it will never happen. There will be outlying "big PRs" for quite some time, and it is not so hard to rebase fixup.